data/nav.yml

```diff
@@ -416,6 +416,11 @@
           path: "apis/rest-api"
         - name: "Limits"
           path: "apis/rest-api/limits"
+        - name: "Organization REST API"
```

Comment:
Did you have any thoughts behind the position of this?

I can see it up here being product-agnostic, but think it probably gives it too much prominence. I'd suggest either adding it below the existing ones to keep the change small. Or, perhaps this is a good time to alphabetize them.

I'd suggest going with alphabetizing:

- Agent REST API
- Analytics REST API
- Pipelines REST API
- Organization REST API

Reply:
Can do!